var/spack/repos/builtin/packages/usalign/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Usalign(Package):
+    """US-align (Universal Structural alignment) is a unified protocol
+    to compare 3D structures of different macromolecules (proteins,
+    RNAs and DNAs) in different forms (monomers, oligomers and
+    heterocomplexes) for both pairwise and multiple structure
+    alignments."""
+
+    homepage = "https://zhanggroup.org/US-align"
+    url = "https://zhanggroup.org/US-align/bin/module/USalign.cpp"
+
+    maintainers("snehring")
+
+    # date assumed from paper publication date
+    version(
+        "20220829",
+        sha256="30274251f4123601af102cf6d4f1a9cc496878c1ae776702f554e2fc25658d7f",
+        expand=False,
+    )
```

Comment:
How did you calculate the sha256?  I'm getting something different:

```suggestion
    version(
        "20220829",
        sha256="9ee129017a68125c22ce89123ecbac9421add87ee077cd1994c6e8a39a8a8b21",
        expand=False,
    )
```

Reply:
Not sure, I'm assuming they've updated the file since I grabbed it Friday. I am getting the same checksum as you now, so I'll change it when I add in a note about the source file.